01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password: ")
+    if not password:
+        get_user_password()
+    return password
```

Comment:
Style: there should be a newline after the if clause:
```
if condition:
    statement()

another_statement()
```
See [Python style guide](https://docs.google.com/document/d/14UPIiU3-X1wSG05wAkatwq2GoioAwXjU5TmUU6J1R10/edit).